queries/ralph/highlights.scm

```diff
@@ -0,0 +1,139 @@
+(type_identifier) @type
+
+(identifier) @variable
+
+[
+  (primitive_type)
+  (alph_token_id)
+] @type.builtin
+
+(var_declaration) @variable
+
+(struct_field
+  (identifier) @variable.member)
+
+(arg_def
+  (identifier) @variable.parameter)
+
+(event_field
+  (identifier) @variable.parameter)
+
+(struct_constructor_field
+  (identifier) @variable)
+
+[
+  (hex_num)
+  (integer)
+  (typed_num)
+] @number
+
+[
+  (bool)
+  (bytes)
+  (address)
+] @constant.builtin
+
+(func_id) @function
+
+(func_id
+  (identifier) @function)
+
+(annotation
+  [
+    "@"
+    (identifier)
+  ] @attribute)
+
+(annotation_field
+  (identifier) @variable)
```

Comment:
The (identifier) in the first few lines should be covering for this already

Reply:
Indeed, I just push a fix.